content/reference/storage/deleting-key-value-pairs.md

```diff
@@ -12,6 +12,23 @@ This will remove the key and value from your namespace. As with any
 operations, it may take some time to see that they key has been deleted from
 various points at the edge.
 
+This method returns a promise that you should `await` on in order to verify
+successful deletion.
+
+Here's an example of deleting a key from within a Worker:
+
+```js
+addEventListener('fetch', event => {
```

Comment:
I would leave out the script context and just say `await KV_NAMESPACE.delete("some-key")`